torch/csrc/autograd/profiler.h

```diff
@@ -195,6 +268,27 @@ struct TORCH_API Event final {
     return handle_;
   }
 
+  // Node ID corresponding to this event.
+  int node_id( ) const {
+    return node_id_;
+  }
+
+  // Set Node ID on this event.
+  void setNodeId(int node_id) {
+    node_id_ = node_id;
+  }
+
+  // Set CPU memory usage on this event
+  Event& setCPUMemoryUsage(int64_t cpu_memory_usage) {
+    cpu_memory_usage_ = cpu_memory_usage;
+    return *this;
+  }
+
+  // Set relative_elapsed_cpu_ns
+  void setRelativeElapsedCpuUs(double relative_elapsed_cpu_us) const {
```

Comment:
why making this const and `relative_elapsed_cpu_us_` mutable?

Reply:
Agreed that this is hacky, fixed in latest version